packages/SwingSet/src/gc-and-finalize.js

```diff
@@ -63,23 +64,23 @@
  * dummy pre-resolved Promise.
  */
 
-let alreadyWarned = false;
-export async function gcAndFinalize() {
-  if (typeof gc !== 'function') {
-    if (!alreadyWarned) {
-      alreadyWarned = true;
-      console.warn(
-        Error(`no gc() function; disabling deterministic finalizers`),
-      );
-    }
-    return;
+export function makeGcAndFinalize(gcPower) {
```

Comment:
I'm kinda glad to see that `alreadyWarned` go away.. I'm not sure how it worked on our unit test (which extracted the source code of `gcAndFinalize` for evaluating in xsnap, which would lose the `let alreadyWarnred = false;` line). Maybe it was just relying upon unbound names being treated as properties of the global.

Reply:
Yeah, I don't know how that worked.